web-server/views/users/profile.jade

```diff
@@ -0,0 +1,31 @@
+extends ../layout
+
+block content
+
+  .row
+    .col-md-4
+      h1= user.name
+      ul.list-inline
+        li
+          a(href=user.profileUrl target="_blank") @#{user.handle}
```

Comment:
I havent been putting an `@` in front of handles anywhere else in this app. I think we should stay consistent and not @ our github handles.